public_html/wp-content/themes/wporg-events-2023/functions.php

```diff
@@ -39,6 +40,41 @@ function enqueue_assets() {
 	);
 }
 
+/**
```

Comment:
There's some existing code that does similar things:

* https://github.com/WordPress/wordcamp.org/blob/72b6a8e6ea6d3bc6dfb4c070ef2a093c5f744110/public_html/wp-content/mu-plugins/jetpack-tweaks/sharing.php
* https://meta.trac.wordpress.org/browser/sites/trunk/wordpress.org/public_html/wp-content/mu-plugins/pub/site-branding.php?rev=11778

Would those work, maybe with some modifications, rather than duplicating it?

Reply:
Ah, nice. I wasn't aware of that code. Yes, we should re-use thanks.